pkg/controller/build/buildrequest/assets/create-digest-cm.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# This script is not meant to be directly executed. Instead, it is embedded
```

Comment:
thought (non-blocking): Another idea that I've been cogitating on is replacing these scripts with a Golang binary that could connect to the API server, fetch and clone everything needed, and then make the API server updates upon completion. An advantage is that we'd have no more ephemeral API objects since any copies or mutations are done locally inside the build pod instead of via the API server.